bluesky/tools/network.py

```diff
@@ -143,5 +170,11 @@ def __init__(self):
     def connect(self, fun):
         self.process = fun
 
-    def processData(self, sender_id, data):
-        self.process(bytearray(data).decode(encoding='ascii', errors='ignore').strip())
+    def processData(self, data, sender_id):
+        msg = bytearray(data).decode(encoding='ascii', errors='ignore').strip()
+
+        if msg.startswith(CMD_TCP_CONNS):
+            self.connections[sender_id].sendReply(
+                str(self.numConnections()))
```

Comment:
Command TCP_CONNS can be used to debug number of active tcp sessions, and simply returns this information.